### DISCUSSION TITLE
SpotBugs ended with exit code 1

### DISCUSSION THREAD

**Discussion by soloturn** [spotbugs/spotbugs-gradle-plugin#1025] [General]
what should a message:
```
> Task :subsystems:TypeHandlerLibrary:spotbugsMain
SpotBugs ended with exit code 1

```
mean, and how to find the reason, or silence it?

**Comment by jica** [comment]
According to [this test](https://github.com/spotbugs/spotbugs-gradle-plugin/blob/20a9ecb72b6e81ae773d5a38521b8acea88295cf/src/functionalTest/groovy/com/github/spotbugs/snom/StandardFunctionalTest.groovy#L260), that message means that there are bugs.
